os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -639,55 +639,69 @@ static void i2s_rxdma_timeout(int argc, uint32_t arg)
  ****************************************************************************/
 
 #ifdef I2S_HAVE_RX
-static int i2s_rxdma_setup(struct s5j_i2s_s *priv)
+
+static int i2s_rxdma_prep(struct s5j_i2s_s *priv, struct s5j_buffer_s *bfcontainer)
```

Comment:
I see s5j_buffer_s (generic buffer name) is used for dma. It would have nice if it had been named s5j_dmabuf_s or similar.

Reply:
It can be done later, though it is not part of this commit.